src/renderer/components/QueryResultTable/QueryResultTable.css

```diff
@@ -11,11 +11,26 @@
 }
 
 .QueryResultTable-table th {
-  border-bottom: 3px double #ccc;
   text-align: left;
   font-weight: bold;
 }
 
+.QueryResultTable-table thead th {
+  top: 46px; /* (height + border-bottom) of QueryResultNav */
+  position: sticky;
+  background-color: white;
+}
+
+/* Use pesudo class to protect from overlapping border */
+.QueryResultTable-table thead th:after {
```

Comment:
If I set border-bottom to `th` tag, it is overlapped by first row.

I find the idea by using pseudo class in this article↓
https://stackoverflow.com/a/50362396/6945346